tests/lib/rules/no-jquery.js

```diff
@@ -187,6 +240,20 @@ eslintTester.run('no-jquery', rule, {
         },
       ],
     },
+    {
+      code: `
+        export default Ember.Component({
+          didInsertElement() {
+            this.$.extend({}, a, b)
+          }
+        });`,
+      output: null,
+      errors: [
+        {
+          message: ERROR_MESSAGE,
```

Comment:
Should also assert the error node `type`.

Reply:
I've added `type` to all the error cases. Hopefully that makes it a little clearer wheat each test is expecting 😅